### DISCUSSION TITLE
e.get is not a function

### DISCUSSION THREAD

**Discussion by stevee-bee** [codefori/vscode-ibmi#947] [IBM i] [answered]
After installing Db2 for IBM i v0.2.6 and clicking the Add Schema button, VS Code tells me "e.get is not a function" and nothing is added to the schema browser. BTW, I have Code for IBM i v1.5.24 installed, connected, and working. What else should I check?

**Comment by worksofliam** [comment, marked as answer]
@stevee-bee Literally working on a PR to fix this right now.

**Reply by worksofliam** [reply]
@stevee-bee vscode-ibmi 1.5.25 is now out which should solve this problem, as well as vscode-db2i 0.2.7.

**Reply by stevee-bee** [reply]
@worksofliam - perfect, thanks!
